pkgs/development/python-modules/pyshark/default.nix

```
@@ -36,7 +34,7 @@ buildPythonPackage rec {
   ];
 
   preCheck = ''
-    export HOME=$TMPDIR
+    export HOME=$(mktemp -d);
```

```suggestion
    export HOME=$(mktemp -d)
```
also it does not really matter. Both achieve the same result